pytype/tools/merge_pyi/merge_pyi.py

```diff
@@ -626,6 +626,12 @@ def insert_comment_annotation(self, src_sig, pyi_sig):
       return []
 
   def can_annotate(self, src_sig):
+    if src_sig.short_name.startswith('__') and \
+            src_sig.short_name.endswith('__') and \
+            src_sig.short_name != "__init__":
+      self.logger.warning('builtin function, skipping %s', src_sig)
```

Comment:
Shouldn't this be info? It'll be happening a lot and it's not an indication of anything going wrong (a warning is indication that something is probably OK but should be checked)

Reply:
Oh, exactly. My bad. Done.